evm/src/main/java/com/horizen/evm/interop/EvmResult.java

```diff
@@ -3,25 +3,20 @@
 import com.horizen.evm.utils.Address;
 
 import java.math.BigInteger;
-import java.util.Map;
 
 public class EvmResult {
     public BigInteger usedGas;
     public String evmError;
     public byte[] returnData;
     public Address contractAddress;
-    public EvmTraceLog[] traceLogs;
-    public Map<String,Integer> fourByteTracerLogs;
-    public EvmCallTraceLog callTracerLogs;
     public Boolean reverted;
+    public Object tracerResult;
```

Comment:
We could also use `JsonNode` here, to clarify that this is unparsed JSON.

Reply:
Datatype changed to the less generic `JsonNode`